src/visuallyCompleteCalculator.ts

```diff
@@ -134,14 +141,6 @@ class VisuallyCompleteCalculator {
       // identify timestamp of last visible change
       const end = Math.max(start, this.lastImageLoadTimestamp, this.lastMutation?.timestamp ?? 0);
 
-      const navigationType = isBfCacheRestore
-        ? 'back_forward'
-        : activationStart > 0
-        ? 'prerender'
-        : start > 0
-        ? 'script'
-        : getNavigationType();
-
```

Comment:
Can we simplify this?

It seems to me this might be the only change we need here:

```diff
     const navigationType = isBfCacheRestore
        ? 'back_forward'
-       : activationStart > 0
+       : activationStart >= start
        ? 'prerender'
        : start > 0
        ? 'script'
        : getNavigationType();
```